src/administration/web/configure-environment.md

```diff
@@ -66,22 +66,25 @@ project configuration screen.
 
 ## Robots.txt
 
-By default, Platform.sh returns a restrictive `robots.txt` on all environments. If you need to provide a custom `robots.txt`, first disable the default one using the [Platform.sh CLI](/overview/cli.md) command below:
+By default, Platform.sh serves the following default and restrictive `robots.txt` on all non-production environments:
 
 ```
-platform environment:info restrict_robots false
+User-agent: *
+Disallow: /
 ```
 
-Or to disable it for a specific environment rather than production, execute the following:
+That tells search engines to ignore pre-production sites entirely, even if they are publicly visible.  To disable that feature for pre-production sites use the [Platform.sh CLI](/overview/cli.md) command below:
 
 ```
-platform environment:info -e ENVNAME restrict_robots false
+platform environment:info restrict_robots false
 ```
 
-(Where `ENVNAME` is the name of the environment, which is generally the branch associated with it.)
+Or to disable it for a specific environment other than the one that is currently checked out, execute the following:
 
-Then you have to serve your `robots.txt` by configuring ["/" location in `.platform.app.yaml`](/configuration/app-container.md#locations).
+```
+platform environment:info -e ENVNAME restrict_robots false
+```
 
-> **note**
-> A deployment is required after running the CLI command to make the changes effective.
+(Where `ENVNAME` is the name of the environment, which is generally the branch associated with it.)  Note that you will need to trigger a new deployment after changing this setting.
```

Comment:
The sentence in parentheses is a bit weird. The `name` or `id` of an environment is always the same as the Git branch name.